### ISSUE TITLE
Some questions about rand_crop=false

### ISSUE THREAD

**Issue by ZHR1997** [xingyizhou/ExtremeNet#44]
When I train my own data, I set rand_crop to false. Then when the trained model is tested on the training set, there is no box on the output image, and the loss at the end of the training is also very small. So it may be due to rand_crop = false, and then we need to change the demo code?
